core/src/main/scala/frameless/CatalystIsin.scala

```diff
@@ -0,0 +1,18 @@
+package frameless
+
+import scala.annotation.implicitNotFound
+
+/** Types for which we can check if is in */
+@implicitNotFound("Cannot do isin operation on columns of type ${A}.")
+trait CatalystIsin[A]
+
+object CatalystIsin {
```

Comment:
Hello, aren’t these the same as anything that is comparable? We already have a typeclass for those.

Reply:
@imarios which typeclass are you referring to ? also CatalystIsin should contain `Array` type but didn't manage to add it

Reply:
Hi @ayoub-benali, that would be CatalystOrdered. I think anything that can be ordered (compared) can also be checked during an "isin".